hub/core/meta/tensor_meta.py

```diff
@@ -37,30 +37,24 @@ class TensorMeta(Meta):
     sample_compression: str
     chunk_compression: str
 
-    @staticmethod
-    def create(
-        key: str,
-        storage: StorageProvider,
+    def __init__(
+        self,
         htype: str = DEFAULT_HTYPE,
         **kwargs,
     ):
         """Tensor metadata is responsible for keeping track of global sample metadata within a tensor.
-
```

Comment:
What happened to these line breaks? They're important for API reference auto-formatting.

Reply:
not sure what you are referring to

Reply:
@mccrearyd the diff shows the separating lines were removed. So we went from
```
Tensor metadata ...

Note:
    ...
```

to

```
Tensor metadata ...
Note:
    ...
```

the separation is necessary for certain formatters/checkers to recognize the different sections.

Reply:
ahh i see